signbank/dictionary/csv_import.py

```diff
@@ -816,6 +816,8 @@ def import_manual_validation(request):
         for row in validation_record_reader:
             if validation_record_reader.line_num == 1:
                 continue
+            if ":" not in row["idgloss"]:
+                continue
```

Comment:
Micky said the times that colons were missing were her manual error so she says this check probably isn't necessary. But I guess it can't hurt?

Reply:
lets get rid of them to avoid possible confusion, since Micky expects this to error